feature/bgp/gracefulrestart/ate_tests/err/README.md

```diff
@@ -0,0 +1,289 @@
+As per the [IETF Draft on LLGR](https://tools.ietf.org/html/draft-ietf-idr-long-lived-gr), we have the following that is different from EER.
+
+  - Section 4.2 / 4.3 of the draft: mandates what communities are in use and what their specific behavior should be. For example: "The "LLGR_STALE" community must be advertised by the GR helper and also MUST NOT be removed by other receiving peers." and anyone that receives that route MUST treat the route as least-preferred. This isnt the case for ERR. There arent any communities attached to Stale routes thereby mandating their depreference.
+  - Section 4.7: Different conditions for partial deployment of LLGR is a no-op for ERR as it builds on the concepts of RFC8538 and hence there arent any special communities expected to be sent or received for the stale routes.
+
+## Testbed type
+[atedut_2.testbed](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+* Test environment setup
+  ## Topology
+Create the following connections:
+```mermaid
+graph LR; 
+A[ATE:Port1] <-- IBGP(ASN100) --> B[Port1:DUT:Port2];
+B <-- EBGP(ASN200) --> C[Port2:ATE];
+```
+* ATE:Port1 runs IBGP and must advertise the following IPv4 and IPv6 prefixes with the corresponding commiunitty attributes
+  * IPv4Prefix1 and IPv6Prefix1 with community NO-ERR
+  * IPv4Prefix2 and IPv6Prefix2 with community ERR-NO-DEPREF
+  * IPv4Prefix3 and IPv6Prefix3 with community TEST-IBGP
+* ATE:Port2 runs EBGP and must advertise the following IPv4 and IPv6 prefixes with the corresponding commiunitty attributes
+  * IPv4Prefix4 and IPv6Prefix4 with community NO-ERR
+  * IPv4Prefix5 and IPv6Prefix5 with community ERR-NO-DEPREF
+  * IPv4Prefix6 and IPv6Prefix6 with community TEST-EBGP
+* DUT has the following configuration on its IBGP and EBGP peering
+  * Extended route retention (ERR) enabled.
+  * ERR configuration has the retention time of 300 secs configured
+  * ERR has a retention-policy `STALE-ROUTE-POLICY` attached. The DUT **MUST** apply this policy on post-policy adj-rib-in routes. Therefore, import export policy applied to the routes must not be overridden by this policy, it MUST instead be additive.
+  * "STALE-ROUTE-POLICY" has policy-statements to
+    * identify routes tagged with community `NO-ERR` and have an action of "REJECT" so such routes aren't considered for ERR but only GR
+    * identify routes tagged with community `ERR-NO-DEPREF` and have an action of "ACCEPT" so such routes are considered for ERR. Also ADD community `STALE` to the existing community list attached as part of the regular adj-rib-in post policy for the route.
+    * Catch-all rule to identify and accept all other prefixes, attach a local-preference of "0" and ADD community `STALE` to the existing community list.
+  * DUT has import-policy importibgp and export-policy exportibgp towards the IBGP neighbor applied in the import and export directions respectively.
```

Comment:
I suspect that it might be easier to just define these policies in OC ;-), since you're basically using vendor-neutral text to be able to define what you need. WDYT?

Reply:
I had it already defined in the Yaml section at the bottom. However, I revisited that to look for missing ones.